CMakeLists.txt

```diff
@@ -330,6 +334,7 @@ if (NOT WIN32)
 endif ()
 
 add_library(freeorioncommon
+    $<$<CXX_COMPILER_ID:msvc>:STATIC>
```

Comment:
Hm, in that case I suspect @adrianbroher changed that to be build as a dynamic lib. It's his goal to turn the common and parser libs into shared libraries after all. So, I wouldn't just change it back to static, at least not without feedback from him - @adrianbroher?